test/custom/BidWhenPaused.test.js

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: MIT
+// ============ External Imports ============
+const { waffle } = require('hardhat');
+const { provider } = waffle;
+const { expect } = require('chai');
+// ============ Internal Imports ============
+const { eth, bidThroughParty, contribute, placeBid } = require('../helpers/utils');
+const { deployTestContractSetup } = require('../helpers/deploy');
+const { MARKET_NAMES } = require('../helpers/constants');
+const { testCases } = require('../testCases.json');
+
+describe('Bid When Paused', async () => {
+  testCases.map((testCase, i) => {
+    describe(`Case ${i}`, async () => {
+      // get test case information
+      const { auctionReservePrice, contributions, bids } = testCase;
+      // instantiate test vars
+      let partyBid, market, auctionId;
+      const marketName = MARKET_NAMES.NOUNS;
+      const signers = provider.getWallets();
+      const tokenId = 95;
+
+      before(async () => {
+        // DEPLOY NFT, MARKET, AND PARTY BID CONTRACTS
+        const contracts = await deployTestContractSetup(
+          marketName,
+          provider,
+          signers[0],
+          tokenId,
+          auctionReservePrice,
+          false,
+          true, // Pause the auction house
```

Comment:
super nit: feels like it would be cleaner / more readable to call `pause` on the market within the test file rather than the deploy - but doesn't rly matter :)

Reply:
didn't address, not important